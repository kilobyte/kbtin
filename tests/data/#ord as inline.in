#if {65==#ord A} {#show 1} #else {#show 0}
#if {64==#ord A} {#show 1} #else {#show 0}
#if {65=#ord} {#show 1} #else {#show 0}
