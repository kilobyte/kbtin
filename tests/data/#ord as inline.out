1
0
#ord: no argument
0
